js/controllers/tab/workflow/WorkflowTabHandler.js

```diff
@@ -54,5 +56,10 @@
 			$.pkp.controllers.tab.workflow.WorkflowTabHandler,
 			$.pkp.controllers.TabHandler);
 
-/** @param {jQuery} $ jQuery closure. */
+	$.pkp.controllers.tab.workflow.WorkflowTabHandler.prototype.dataChangedHandler_ =
+		function (callingElement, event) {
```

Comment:
The indentation here will probably cause a JS linting error... It looks like the Travis tests, which include linting, didn't run -- you'll need to add a specially-formulated commit to tell the checkout script to get your `pkp-lib` fork, or else it'll use the main one. Make sure that the last commit on your OJS branch contains only the `lib/pkp` submodule update, and the commit message contains `##withanage/stable-3_1_2##` in it.